server/plugin/api.go

```diff
@@ -1225,19 +1256,13 @@ func (p *Plugin) getRepositories(c *UserContext, w http.ResponseWriter, r *http.
 			}
 			opt.Page = resp.NextPage
 		}
-	} else {
-		for {
-			repos, resp, err := githubClient.Repositories.ListByOrg(c.Ctx, org, &github.RepositoryListByOrgOptions{Sort: "full_name", ListOptions: opt})
+
+		if invalidOrgName {
```

Comment:
Can we make it more clear why in this case list of repos from that user is fetched?

Reply:
Maybe name this var `shouldFetchUserReposInsteadOfOrg`